crypto-wasm/src/lib.rs

```diff
@@ -166,6 +174,12 @@ impl From<Bip32PublicKey> for core_crypto::Bip32PublicKey {
     }
 }
 
+impl AsRef<core_crypto::Bip32PublicKey> for Bip32PublicKey {
+    fn as_ref(&self) -> &core_crypto::Bip32PublicKey {
```

Comment:
I'll make `cddl-codegen` spit these out for all types since they're a lot more needed now for us. Before we could do `foo.0` to avoid cloning just to read a wasm parameter's inner rust value, but now they can be in other modules and we can't `pub` it as that messes wasm_bindgen as it wants to expose it to wasm too. This should be cleaner than doing `.0` too.